From 53c7046933d1b622a7f99e70a08a0bf46e38369f Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 11 Jan 2014 12:55:59 +0100
Subject: [PATCH] Makefile: Use elm_prefs_cc and elementary_codegen defined in
 configure

* using binaries from top_buildir doesn't work when cross-compiling elementary
  where we need to pass native binaries in configure params

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 data/objects/Makefile.am | 2 +-
 doc/widgets/Makefile.am  | 2 +-
 src/examples/Makefile.am | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/data/objects/Makefile.am b/data/objects/Makefile.am
index c6a775d..88c4f1a 100644
--- a/data/objects/Makefile.am
+++ b/data/objects/Makefile.am
@@ -11,7 +11,7 @@ filesdir = $(datadir)/elementary/objects
 
 files_DATA = test.edj test_external.edj multip.edj cursors.edj font_preview.edj postit_ent.edj multibuttonentry.edj test_prefs.edj test_prefs.epb test_focus_custom.edj
 
-ELM_PREFS_CC = $(top_builddir)/src/bin/@ELM_PREFS_CC_PRG@
+ELM_PREFS_CC = @elm_prefs_cc@
 
 EXTRA_DIST = \
 test.edc \
diff --git a/doc/widgets/Makefile.am b/doc/widgets/Makefile.am
index c1508b8..c0544f9 100644
--- a/doc/widgets/Makefile.am
+++ b/doc/widgets/Makefile.am
@@ -19,7 +19,7 @@ AM_CPPFLAGS = \
 @ELEMENTARY_EWEATHER_CFLAGS@ \
 @ELEMENTARY_EMAP_CFLAGS@
 
-ELM_PREFS_CC = $(top_builddir)/src/bin/@ELM_PREFS_CC_PRG@
+ELM_PREFS_CC = @elm_prefs_cc@
 
 noinst_DATA = widget_preview_prefs.epb
 
diff --git a/src/examples/Makefile.am b/src/examples/Makefile.am
index 7184ca1..ce80a46 100644
--- a/src/examples/Makefile.am
+++ b/src/examples/Makefile.am
@@ -33,9 +33,9 @@ EDJE_FLAGS_VERBOSE_0 =
 EDJE_FLAGS_VERBOSE_1 = -v
 EDJE_FLAGS = $(EDJE_FLAGS_VERBOSE_$(V))
 
-ELM_PREFS_CC = $(top_builddir)/src/bin/@ELM_PREFS_CC_PRG@
+ELM_PREFS_CC = @elm_prefs_cc@
 
-ELEMENTARY_CODEGEN = $(top_builddir)/src/bin/@ELEMENTARY_CODEGEN_PRG@
+ELEMENTARY_CODEGEN = @elementary_codegen@
 ELEMENTARY_CODEGEN_FLAGS = --prefix=codegen_example
 
 SRCS = \
-- 
1.8.5.2

